### PR TITLE
added folder icons

### PR DESCRIPTION
**Pull request by yacut**

### CODE REVIEW

**Comment by ivogabe** [comment]
Can you add this behind an option, similar to the switch between Ionicons and Devicons (ec3244c1cc63b6c267689f8a5b7bc04a4c11c551)? The folder icon should also be dependent of the icon pack that is used, so when using Devicons it should show an icon of that pack.

File: lib/main.ts [hunk 1]
```diff
@@ -72,15 +72,21 @@ const createIcon = (data: Icon, secondary: boolean) => {
 };
 
 const provider = (entry) => {
+    var data;
```
Comment (ivogabe) [inline lib/main.ts]: Add a type annotation here (I guess `Icon[]`) and use `let` instead of `var`.

File: lib/main.ts [hunk 2]
```diff
@@ -72,15 +72,21 @@ const createIcon = (data: Icon, secondary: boolean) => {
...
 	if (!entry.isFile) {
-		return;
+		data = [{
+                color: "#666",
```
Comment (ivogabe) [inline lib/main.ts]: Indentation seems to be wrong here, can you replace the spaces with tabs?

**Comment by yacut** [comment]
> The folder icon should also be dependent of the icon pack that is used, so when using Devicons it should show an icon of that pack.

There is no 'Devicons' folder icon. The other issues are done.

**Comment by ivogabe** [comment]
Thanks!

**Pull request merged**